microsoft-edge/webdriver-chromium/ie-mode.md

```diff
@@ -0,0 +1,222 @@
+
+*  `dir`, which is the directory where you downloaded IEDriver to.
+*  `driverexe`, which is the name of the IEDriver executable.
+
+```csharp
+//change the path accordingly
+var dir = @"C:\<path to IEDriver download>";
+var driverexe = "IEDriverServer.exe";
+
+var ieService = InternetExplorerDriverService.CreateDefaultService(dir, driverexe);
+```
+
+### [Python](#tab/python/)
+
+<a id="drive-ie-mode-ieService"></a>
+
+<!-- todo -->
+Insert Python sample here
+
+* * *
+
+
+<!-- ====================================================================== -->
+## Define InternetExplorerOptions with additional capabilities for Microsoft Edge
+
+### [C#](#tab/c-sharp/)
+
+<a id="drive-ie-mode-ieOptions"></a>
+
+1. Define a new variable, `ieOptions`, by instantiating `InternetExplorerOptions` with the `IgnoreZoomLevel` property set to `true`.
+
+1. Call the `AddAdditionalCapability` method, with the `ie.edgechromium` property set to `true`, and `ie.edgepath` set to the path of the Microsoft Edge executable:
+
+```csharp
+var ieOptions = new InternetExplorerOptions { IgnoreZoomLevel = true };
+ieOptions.AddAdditionalCapability("ie.edgechromium", true);
+//change the path accordingly
+ieOptions.AddAdditionalCapability("ie.edgepath", 
+       "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe");
+```
+
+### [Python](#tab/python/)
+
+<a id="drive-ie-mode-ieOptions"></a>
+
+<!-- todo -->
+Insert Python sample here
+
+* * *
+
+
+<!-- ====================================================================== -->
+## Start InternetExplorerDriver
+
+### [C#](#tab/c-sharp/)
+
+<a id="drive-ie-mode-ieDriver"></a>
+
+Finally, start `InternetExplorerDriver` and pass it the previously defined `ieService` and `ieOptions`.  IEDriver launches Microsoft Edge in IE mode.  All page navigation and subsequent interactions occur in IE mode.
+
+```csharp
+InternetExplorerDriver reqDriver = new InternetExplorerDriver(ieService, ieOptions);
+```
+
+### [Python](#tab/python/)
+
+<a id="drive-ie-mode-ieDriver"></a>
+
+<!-- todo -->
+Insert Python sample here
+
+* * *
+
+
+<!-- ====================================================================== -->
+## Known limitations
+
+When automating IE mode in Microsoft Edge, you must add a short wait between opening a new window with [window.open]https://developer.mozilla.org/docs/Web/API/Window/open) and getting handles to the new window with the [Get Window Handles](https://www.w3.org/TR/webdriver2/#get-window-handles) command.
+
+The following sample demonstrates how you must wait for IEDriver to register new window handles when opening new windows.
+
+### [C#](#tab/c-sharp/)
+
+<a id="drive-ie-mode-window-handlers"></a>
+
+After the `Click` method is called on a button that opens a new window, IEDriver must wait 2 seconds with `Thread.Sleep(2000)` before getting new window handles with `reqDriver.WindowHandles`.
```

Comment:
Does this sample really not work without the wait? If so, that's surprising and might indicate a bug in our fix.